src/Sculpin/Bundle/PaginationBundle/PaginationGenerator.php

```diff
@@ -166,4 +172,44 @@ public function generate(SourceInterface $source)
 
         return $sources;
     }
+
+    /**
+     * @param $matches
+     *
+     * @return array
+     */
+    private function filtered($matches)
+    {
+        list($name, $expectedKey, $expectedValue) = explode('.', $matches[2]);
+
+        if ($expectedValue === 'true') {
+            $expectedValue = '1';
+        }
+
+        if ($expectedValue === 'false') {
+            $expectedValue = '';
+        }
+
+        $data = $this->dataProviderManager->dataProvider($name)->provideData();
+        if (!count($data)) {
+            $data = array();
```

Comment:
How about using the short array syntax? I think this short array syntax is better than `array` :smile:.